eng/templates/azure-pipelines-shared-jobs.yml

```diff
@@ -0,0 +1,134 @@
+parameters:
+- name: isOfficial
+  displayName: Indicates whether to use official or unofficial 1ES templates
```

Comment:
I'm not sure we want to tie this to the 1ES templates. Maybe having it be `dev` versus `prod` or `official` even would be better? Highlighting the intent rather than the implementation details